tests/test_todo.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import requests
+
+
+@pytest.mark.asyncio
+async def test_all_todo():
+    response = requests.get("http://localhost:8000/api/")
+    assert response.status_code == 200
+    assert response.text == '"Not Yet"'
+
+
+@pytest.mark.asyncio
+async def test_post_todo():
+    response = requests.post("http://localhost:8000/api/")
+    assert response.status_code == 200
+    assert response.text == '"Not Yet"'
+
+
+@pytest.mark.asyncio
+async def test_put_todo():
+    response = requests.put("http://localhost:8000/api/1")
+    assert response.status_code == 200
+    assert response.text == '"Not Yet"'
+
+
+@pytest.mark.asyncio
+async def test_delete_todo():
+    response = requests.delete("http://localhost:8000/api/1")
```

Comment:
`requests` is a synchronous library. No need for `async def`. In general, you need async functions when you have an `await` keyword inside. Otherwise, use a different HTTP client.